docs/policygenerator-reference.yaml

```
@@ -60,6 +60,15 @@ policyDefaults:
   # annotation. This defaults to ["NIST SP 800-53"].
   standards:
     - "NIST SP 800-53"
+  # Optional. Array of policysets that policy is going to be part of. policyset details can be defined
+  # in policysets section. When a policy is part of a policyset, placementbinding will no longer be 
+  # generated for policy. Set policies[*].generatePolicyPlacement or policyDefaults.generatePolicyPlacement
+  # to override.
+  policysets: []
+  # Optional. When a policy is part of a policy set, by default the generator will no longer generate the placement
+  # for this policy. If a placement should still be generated, set it to "true". This mean this policy will be deployed
+  # with both policy placement and policyset placement. This defaults to "false".
+  generatePlacement: false
```

I find the name of this field confusing. I think a more specific name like `generatePlacementWithSet` would be clearer.
```suggestion
  # Optional. When a policy is part of a policy set, by default the generator will not generate the placement
  # for this policy since a placement is generated for the set. If a placement should still be generated, set it to "true" so that this policy will be deployed
  # with both policy placement and policy set placement. This defaults to "false".
  generatePlacementWithSet: false
```

---

I like `generatePlacementWhenInSet`